src/utils.js

```diff
@@ -530,6 +530,18 @@ go.utils = {
             });
     },
 
+    resend_all_subscriptions: function(im, identity) {
+        return go.utils
+            .get_active_subscriptions_by_identity(im, identity.id)
+            .then(function(subscriptions){
+                for (var i=0; i<subscriptions.length; i++) {
+                    subscription = subscriptions[i];
+                    var endpoint = 'subscriptions/' + subscription.id + '/resend';
+                    go.utils.service_api_call('subscriptions', 'post', {}, {}, endpoint, im);
```

Comment:
I think this is an async method? So we probably want to collect all the calls and then yield on them.

We can probably use a combination of `_.map` (https://lodash.com/docs/4.17.4#map), which can create the list of promises for us, and then using `Promise.all` (https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise/all) to combine them into a single promise to return.